lib/getFeatures.js

```diff
@@ -67,13 +71,18 @@ export async function fixedGetFeatures(options) {
  */
 function findTemplates(features) {
     return features.map((feature, fIndex) => {
+        let background;
+        if (feature.scenarios[0].steps[0].isBackground) {
```

Comment:
what if feature does not have steps?

Reply:
This code works fine, because feature without steps isn't get from the first place  (in getFeatures() function). I've double checked it.